physics/SFC_Models/Land/Noahmp/module_sf_noahmplsm.F90

```diff
@@ -5607,6 +5630,11 @@ subroutine gfs_stability                                              &
 real(kind=kind_phys) :: zolmax
 
 real(kind=kind_phys) xkzo
+!   tofd
+      real(kind=kind_phys) :: cf, zf, fri, fphim
```

Comment:
line up with the rest of the declarations

Reply:
resolved